pkg/gds/certs.go

```diff
@@ -85,7 +85,25 @@ func (s *Service) HandleCertifcateRequests(certDir string) (err error) {
 			wg.Add(1)
 			go func(logctx zerolog.Logger) {
 				defer wg.Done()
-				if err := s.submitCertificateRequest(req); err != nil {
+				// Get the VASP from the certificate request
+				var vasp *pb.VASP
+				if vasp, err = s.db.RetrieveVASP(req.Vasp); err != nil {
+					logctx.Error().Str("vasp_id", req.Vasp).Msg("could not retrieve vasp for certificate request submission")
+					return
+				}
+
+				// Verify that the VASP has not errored or been rejected
+				if vasp.VerificationStatus != pb.VerificationState_REVIEWED {
```

Comment:
I think this needs to be `vasp.VerificationStatus < pb.VerificationState_REVIEWED  || vasp.VerificationStatus > pb.VerificationState_VERIFIED`  -- because when we reissue certificates the VASP will be in the verified state. The `>` and `<` operators will also include `pb.VerificationState_ ISSUING_CERTIFICATE `  but that seems ok to me.